tests/main/test_timestamp_featurizer.py

```diff
@@ -23,9 +23,9 @@ def test_no_timestamps(self, iris):
         featurizer = CustomTimestampFeaturizer(iris[DatasetConstants.FEATURES]).fit(x_train)
         result = featurizer.transform(x_test)
         # Assert result is same as before, pandas dataframe
-        assert(isinstance(result, pd.DataFrame))
+        assert (isinstance(result, pd.DataFrame))
```

Comment:
nit: I think we can remove quotes here as well on these asserts

Reply:
done